src/profiles/services/proof.ts

```diff
@@ -0,0 +1,6 @@
+export interface Proof {
+  proof_url: string;
```

Comment:
Re modelling, this could be "proofUrl".

Reply:
True, not sure why the original code was using underscores. Is there a reason `@typescript-eslint/camelcase` is turned off?